### PR TITLE
test: add playwright tests

### PR DESCRIPTION
**Pull request by roope-ankka**
## What?
This commit adds additional Playwright tests

## How to test

Run the tests locally with `yarn test:e2e:start` and verify that all tests pass ✅

### CODE REVIEW

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HAUKI-ADMIN-UI branch is deployed to platta: https://hauki-pr395-admin-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HAUKI-ADMIN-UI branch is deployed to platta: https://hauki-pr395-admin-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HAUKI-ADMIN-UI branch is deployed to platta: https://hauki-pr395-admin-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://hauki-pr395-admin-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://hauki-pr395-admin-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HAUKI-ADMIN-UI branch is deployed to platta: https://hauki-pr395-admin-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://hauki-pr395-admin-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: e2e/tests/resource-page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+import { Page, expect, test } from '@playwright/test';
+import { getResource, getResourceUrl } from '../utils';
+import { Resource } from '../../src/common/lib/types';
+
+test.describe('Resource page', async () => {
+  let page: Page;
+  let resourceUrl: string;
+  let resource: Resource;
+
+  const haukiUser = process.env.HAUKI_USER || '?';
```
Comment (mikkojamG) [inline e2e/tests/resource-page.spec.ts:11]: Could maybe default to `cypress_test_user` or something equivalent? 🤔

File: e2e/utils.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { promisify } from 'util';
+import { exec } from 'child_process';
+import { DatePeriod, Resource } from '../src/common/lib/types';
+
+const apiUrl = process.env.API_URL || 'http://localhost:8000';
```
Comment (mikkojamG) [inline e2e/utils.ts:5]: Could maybe default to `https://hauki-api.dev.hel.ninja/` 🤔

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://hauki-pr395-admin-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HAUKI-ADMIN-UI branch is deployed to platta: https://hauki-pr395-admin-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://hauki-pr395-admin-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: e2e/tests/resource-page.spec.ts (hunk 1 above)
Comment (roope-ankka) [reply e2e/tests/resource-page.spec.ts]: Previously setting the correct HAUKI parameters for the e2e-tests was assigned to the one running the tests 👎🏻 Now the required variables are defined within the tests 👍🏻

`const testData = {
   HAUKI_USER: 'cypress_test_user',
   HAUKI_ORGANIZATION: 'test:cypress',
   HAUKI_SOURCE: 'test',
   HAUKI_RESOURCE: 'test:1',
 };`

File: e2e/utils.ts (hunk 2 above)
Comment (roope-ankka) [reply e2e/utils.ts]: Done ✅

**Review by mikkojamG — APPROVED** [review]

**Pull request merged**